src/proj.rs

```diff
@@ -1018,6 +1018,8 @@ impl convert::TryFrom<(&str, &str)> for Proj {
     }
 }
 
+/// Info about the current PROJ definition
+#[derive(Copy, Clone, Debug)]
 struct PjInfo {
```

Comment:
```suggestion
pub struct PjInfo {
```

:wink:

Also, I don't think it can be `Copy` - only `Clone` - because of the `String` fields.

Reply:
🤦 night-PR strikes again